src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Utilities;
+using Python.Runtime;
+
+namespace DSCPython
+{
+    [SupressImportIntoVM]
```

Comment:
is this class actually imported into the VM at all?

Reply:
I'm not sure about this. Maybe @reddyashish ?

Reply:
Yes, the class is imported in the VM but not these internal variables. Similar to what is done in IronPythonEvaluator class.